composer.json

```diff
@@ -1,7 +1,11 @@
 {
-  "name": "stvyang-i/indodana-paylater-php-sdk",
+  "name": "indodana/indodana-paylater-sdk-php",
```

Comment:
I think you don't need the `php` here, it's already under php package manager

Reply:
I don't have a strong opinion for (FYI Amazon & Stripe includes `php`, while Facebook are not)

Reply:
I don't have strong opinion either, indodana-paylater-sdk-php is good enough though, alright let me rename the repo